lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -53,12 +53,12 @@ def read
           end
 
           def handle_read(buffer)
-            logger.debug("#{self.class}##{__method__}") { buffer }
+            logger.debug("#{self.class}##{__method__}") { buffer.force_encoding('UTF-8') }
```

Comment:
I think it is better to use `scrub` (if it is available on all supported ruby versions).

Or possibly even `begin` & `rescue` so the logging can't crash reading/writing to a socket.